core/templates/candidate/signup.html

```diff
@@ -158,75 +158,238 @@ <h2 class="title">Registration Form</h2>
+                    console.log("hi")
+                    document.getElementById('university_err').innerHTML = "University roll cannot be empty";
+                    $( "#university_err" ).show();
+                    submit = false;
+                }
+        });
 
-        var Contact = document.getElementById('phoneInput').value;
-        if (Contact == "") {
-            document.getElementById('contactno_err').innerHTML = "Contact cannot be empty";
-            return false;
-        }
-        else {
-            var re = /^(?:(?:\+|0{0,2})91(\s*[\ -]\s*)?|[0]?)?[6789]\d{9}|(\d[ -]?){10}\d$/;
-            if (!re.test(Contact)) {
-                document.getElementById('contactno_err').innerHTML = "Contact number format invalid";
-                return false;
+        $("#university_roll_noInput").focusin(function (event) {
+            if (event.charCode!=0) {
+                $( "#university_err" ).hide();
+                submit = true;
+                document.getElementById('university_err').innerHTML == "";
             }
-        }
-        }
-        catch{
-        // do nothing
+        });
 
-        }
+        $("#id_submit").click(function (e) {
+             if(submit==false ||
+                ($("#name_err").val() != "") ||
+                ($("#email_err").val() != "") ||
+                ($("#contactno_err").val() != "") ||
+                ($("#university_err").val() != "") ||
+                ($("#nameInput1").val() == "") ||
+                ($("#exampleInputEmail1").val() == "") ||
+                ($("#phoneInput").val() == "") ||
+                ($("#university_roll_noInput").val() == "")
+             ){
+                e.preventDefault();
+                console.log("working")
+                $("#id_submit").css('cursor', 'not-allowed');
+             }
+             else{
+                $("#id_submit").css('cursor', 'pointer');
+             }
+             });
+
+        $("#id_submit").hover(function (e) {
+             if(submit==false ||
+                ($("#name_err").val() != "") ||
+                ($("#email_err").val() != "") ||
+                ($("#contactno_err").val() != "") ||
+                ($("#university_err").val() != "") ||
+                ($("#nameInput1").val() == "") ||
+                ($("#exampleInputEmail1").val() == "") ||
+                ($("#phoneInput").val() == "") ||
+                ($("#university_roll_noInput").val() == "")
+             ){
+                e.preventDefault();
+                console.log("working")
+                $("#id_submit").css('cursor', 'not-allowed');
+             }
+             else{
+                $("#id_submit").css('cursor', 'pointer');
+             }
+             });
+
+    });
+    function validateForm() {
+<!--        var name = document.getElementById('nameInput1').value;-->
+<!--        if (name == "") {-->
+<!--            document.getElementById('name_err').innerHTML = "Name cannot be empty";-->
```

Comment:
document.getElementById('name_err') == $("#name_err"). Try to keep your code clean by not mixing various approach to achieve same result. Keep it uniform.